ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -780,6 +814,8 @@ export class WalletDB {
     nullifier: Buffer,
     tx?: IDatabaseTransaction,
   ): Promise<void> {
+    // TODO: maybe keep a count of how many times this method was called, and
+    // reset the bloom filter after a certain threshold
```

Comment:
I think that nullifiers are deleted so infrequently that it might not be worth the added complexity to reset the bloom filter

Reply:
Agreed, that's why I did not implement it. I might turn this "TODO" into a simple comment saying that the bloom filter is unchanged after a delete, and that's by design, because \<explanation\>.